### PR TITLE
Add PID to pipe names

### PR DESCRIPTION
**Pull request by rhatdan**
Allow running of multiple container-storage-setup.sh at the same time.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: container-storage-setup.sh [hunk 1]
```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/css-fifo1
-PIPE2=/run/css-fifo2
+PIPE1=/run/css-$$-fifo1
+PIPE2=/run/css-$$-fifo2
```
Comment (rhvgoyal) [inline container-storage-setup.sh]: So we keep on creating those pipes and never clean these up? Given we can have many more pipes now, I think it is a good idea to remove these once we are done?

Comment (rhatdan) [inline container-storage-setup.sh]: Ok I now remove the pipes.

Comment (rhvgoyal) [inline container-storage-setup.sh]: Sorry, I just noticed that we already have logic to cleanup pipes.

trap cleanup_pipes EXIT
cleanup_pipes(){
    rm -f $PIPE1
    rm -f $PIPE2
    rmdir $TEMPDIR 2>/dev/null
}

So you can get rid of newly added code and push patch again.

Comment (rhatdan) [inline container-storage-setup.sh]: Ok

**Comment by rhvgoyal** [comment]
LGTM

**Comment by rhvgoyal** [comment]
@rh-atomic-bot r+ cc905c3

**Comment by rhvgoyal** [comment]
@rh-atomic-bot r+ cc905c3

**Comment by rhvgoyal** [comment]
@jlebon Is rh-atomic-bot broken? Does not seem to be merging PRs.

**Comment by rhatdan** [comment]
@rh-atomic-bot r=rhvgoyal  cc905c3

**Comment by rhatdan** [comment]
Looks like it is broken.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ cc905c3

**Comment by jlebon** [comment]
@rh-atomic-bot r+ cc905c3

**Comment by jlebon** [comment]
It's due to the repo being renamed. Will be fixed soon.

**Comment by rhvgoyal** [comment]
@jlebon, thanks for looking into it. Waiting for it to get working again.

**Comment by jlebon** [comment]
@rh-atomic-bot r=rhvgoyal cc905c3

**Comment by jlebon** [comment]
@rh-atomic-bot r=rhvgoyal cc905c3

**Comment by jlebon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**